src/pages/en/core-concepts/astro-components.md

```diff
@@ -111,6 +111,10 @@ const myFavoritePokemon = [/* ... */];
 
 You can define local JavaScript variables inside of the frontmatter component script within an Astro component. You can then inject these variables into the component's HTML template using JSX-like expressions!
 
+:::note[dynamic vs reactive]
+Astro allows you to include **dynamic values** (set as a variable's value once, as defined at build time), but **not reactive values** (regularly updated in response to a change in a variable's value). See below for more examples of [differences between Astro and JSX](#differences-between-astro-and-jsx).
+:::
```

Comment:
```suggestion
Using this approach, you can include dynamic values that are calculated in the frontmatter.  But once included, the values will never change, because Astro components are templates that run once. 

See below for more examples of [differences between Astro and JSX](#differences-between-astro-and-jsx).
:::
```